pyof/utils.py

```diff
@@ -0,0 +1,37 @@
+"""General Unpack utils for python-openflow."""
+import pyof.v0x01.common.header
+import pyof.v0x01.common.utils
+import pyof.v0x04.common.header
+import pyof.v0x04.common.utils
```

Comment:
@erickvermot you are not using this imported modules anywhere, so you should not import them here.
Considering your code, I would do:
```python
from pyof import v0x01
from pyof import v0x04

pyof_version_libs = {0x01: v0x01,
                     0x04: v0x04}
```
Note that I'm using the `from` statement, instead of the `import` statement.
This is because on some tests I've performed related to the python importing system, this `import` statement appeared to be less error prone than using the `from ... import` statement, so I would give preference to use the latter.